soh/.clang-tidy

```diff
@@ -1,5 +0,0 @@
-Checks: '-*,readability-braces-around-statements'
-WarningsAsErrors: ''
-HeaderFilterRegex: '(src|include)\/.*\.h$'
-FormatStyle: 'file'
-CheckOptions:
```

Comment:
anyone who has ever made a PR to LUS knows `clang-tidy` is annoying to deal with in CI. unless we find a way to make that process painless i think we should stick with just using `clang-format`